health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -1611,13 +1885,211 @@
+
+    // Return both the updated boundary data and latLongData
+    return {
+        updatedData: boundaryData,
+        latLongData
+    };
+}
+
+async function autoGenerateBoundaryCodesForGeoJson(request: any, localizationMap?: any) {
+    const { hierarchyType, tenantId } = request?.body?.ResourceDetails || {};
+    // const type = request?.body?.ResourceDetails?.type;
+    const fileResponse = await httpRequest(config.host.filestore + config.paths.filestore + "/url", {}, { tenantId, fileStoreIds: request?.body?.ResourceDetails?.fileStoreId }, "get");
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)